synapse/storage/schema/state/delta/30/state_stream.sql

```diff
@@ -26,6 +26,10 @@
  * (event, state) pair, we can use that stream_ordering to identify when
  * the new state was assigned for the event.
  */
+
+/* NB: This table belongs to the `main` logical database; it should not be present
+ * in `state`.
+ */
```

Comment:
We should make a PR to remove this file?

Reply:
Do we also want to DROP TABLE IF EXISTS `ex_outlier_stream` in the state db?

I guess we can't actually do this because some people might be hosting the state and main logical DBs on the same DB cluster.

Reply:
#13788